docs/tutorial.rst

```diff
@@ -3,6 +3,113 @@
 ========
 Tutorial
 ========
+There are two main ways of accessing the resources of the ``stdpopsim`` package
+that will be detailed in this tutorial. The first is via the command line
+interface (CLI). This is useful if you want to do a straightforward run of the
+models in the ``stdpopsim`` :ref:`catalog <sec_catalog>`. The other way to
+access the ``stdpopsim`` resources is via the python API. This is a bit more
+complicated however it allows for more advanced tasks. This tutorial will walk
+through both ways of using the ``stdpopsim`` package as well as some more
+advanced tasks you may wish to do.
+
+***************************
+Using the ``stdpopsim`` CLI
+***************************
+In order to use the ``stdpopsim`` CLI the ``stdpopsim`` package must be
+installed (see :ref:`Installation <sec_installation>`). The CLI provides access
+to the :ref:`catalog <sec_catalog>` of models that have already been implemented
+by ``stdpopsim``. The first step for using the CLI is to select the species that
+you are interested in simulating data for. In order to see which species are
+available run
+
+.. code-block:: console
+
+    $ stdpopsim --help
+
+This shows the species currently supported by ``stdpopsim``. This means that
+``stdpopsim`` knows various traits of these species including chromosome size
+and recombination rates. Once we've selected a species, in this case humans, we
+can look at the help again as follows.
+
+.. code-block:: console
+
+    $ stdpopsim homsap --help
+
+For conciseness we do not show the output here but this time you should see a
+different output which shows options for performing the simulation itself and
+the species default parameters. This includes selecting the demographic model,
+chromosome, recombination map, and number of samples. 
+
+The most basic simulation we can run is to draw two samples using the species'
+defaults as seen in the species help (``stdpopsim homsap --help``). These
+defaults include constant size population, uniform recombination map, and a
+default chromosome. To save time we will also specify that the simulation use
+chromosome 22 with the ``-c`` option. 
+
+.. code-block:: console
+
+    $ stdpopsim homsap -c chr22 -o foo.ts 2
+
+.. warning:: It's important to remember to either redirect the output of ``stdpopsim``
+                to file or to use the ``-o/--output`` option. If you do not, the 
+                binary output may mess up your terminal session.
+
+Say we want to use a specific demographic model. We look up the available models
+using the ``--help-models`` flag:
+
+.. code-block:: console
+
+    $ stdpopsim homsap --help-models
+
+This gives all of the possible demographic models we could simulate. We choose
+the the two population out-of-Africa model from `Tennesen et al.
+<https://doi.org/10.1126/ science.1219240>`_ . By looking at the model help we
```

Comment:
V minor point but I'd add the year of the citation: 2012 Tennesen et al.
Also, I don't think this is a working URL? Remove the space before `science.*` and it's fine I think

Reply:
good catch!